qtconsole/completion_widget.py

```diff
@@ -85,16 +56,24 @@ def eventFilter(self, obj, event):
                 else:
                     self.cancel_completion()
                     if target:
-                        target.event(event)
+                        try:
+                            target.event(event)
+                        except RuntimeError:
+                            pass
```

Comment:
Yes, things work correctly without that block. If they also work for you, you should remove it :-)